app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -202,6 +202,47 @@
 </details>
 
 <script>
+
+ document.getElementById('tab-reporter-' + <%= feedback_message.id %>).addEventListener('click', function (event) {
+   event.preventDefault();
+   var reportId = <%= feedback_message.id %>;
+   showTabContent("reporter", reportId);
+ });
+
+ document.getElementById('tab-offender-' + <%= feedback_message.id %>).addEventListener('click', function (event) {
+   event.preventDefault();
+   var reportId = <%= feedback_message.id %>;
+   showTabContent("offender", reportId);
+ });
+
+ document.getElementById('tab-affected-' + <%= feedback_message.id %>).addEventListener('click', function (event) {
+   event.preventDefault();
+   var reportId = <%= feedback_message.id %>;
+   showTabContent("affected", reportId);
+ });
+
+ function showTabContent(tabName, reportId) {
+   // Make all tabs inactive
+   document.getElementById('tab-reporter-' + reportId).classList.remove("crayons-navigation__item--current");
+   document.getElementById('tab-offender-' + reportId).classList.remove("crayons-navigation__item--current");
+   document.getElementById('tab-affected-' + reportId).classList.remove("crayons-navigation__item--current");
+
+   // Hide all tab related contents
+   document.getElementById('reporter-' + reportId).classList.add("hidden");
+   document.getElementById('offender-' + reportId).classList.add("hidden");
+   document.getElementById('affected-' + reportId).classList.add("hidden");
+   
+   // Remove tab-pane class from all tab contents
+   document.getElementById('reporter-' + reportId).classList.remove("tab-pane");
+   document.getElementById('offender-' + reportId).classList.remove("tab-pane");
+   document.getElementById('affected-' + reportId).classList.remove("tab-pane");
+   
+   // Finally, make the tab active and its related content visible
+   document.getElementById(tabName+ '-' + reportId).classList.remove("hidden");
+   document.getElementById(tabName+ '-' + reportId).classList.add("tab-pane");
+   document.getElementById('tab-' + tabName + '-' + reportId).classList.add("crayons-navigation__item--current");
```

Comment:
**Non-blocking suggestion:** I feel like interpolation is a bit more readable in this case, but the other (simpler) interpolations above ([L226-L238](https://github.com/forem/forem/pull/18924/files#diff-d0e5d2c0c446a2d85849f5a0752e832b87d2703217a8f839d6791c26b88002f0R226-R238)) would also have to change to match these too (so they're consistent). I don't feel too strongly about this so I'm making it non-blocking in case you agree and want to change these

```suggestion
   document.getElementById(`${tabName}-${reportId}`).classList.remove("hidden");
   document.getElementById(`${tabName}-${reportId}`).classList.add("tab-pane");
   document.getElementById(`tab-${tabName}-${reportId}`).classList.add("crayons-navigation__item--current");
```

Reply:
I have applied these changes as they do make code more readable.